### ISSUE TITLE
[Localization] - French

### ISSUE THREAD

**Issue by Cervoz** [XITRIX/Moonlight-Switch#50]
I could help too for a french localization. My POEditor name is Cervoz.

Thx !

**Comment by Cervoz** [comment]
Thank you!

Normally it's fine, everything is translated. How to know if some entries are not too long?

**Comment by XITRIX** [comment]
I could add this localization, compile app and you'll check if everything is fine

**Comment by Vaddum** [comment]
Hi all!

@Cervoz Pas mal du tout ta traduction ! Par contre, si je peux me permettre : 

- Pourquoi as-tu mis des under_score plutôt que des espaces entre les mots ? ;
- Dans les paramètres : pour "Titre_du_mapping des touches" j'aurai simplement mis "Mapping des touches" et pour "Mapping_des_touches_par_défaut" j'aurai mis "Par défaut".
- Quand on synchronise pour la première fois Moonlight avec GeForce Experience, on doit saisir un code à 4 chiffres ; la traduction est bizarre, le code se retrouve écrit au milieu de 2 phrases identiques.

Ce ne sont que des suggestions d'amélioration  ;-) Merci pour ton boulot !

**Comment by Cervoz** [comment]
@XITRIX Ok thanks, I check !

@Vaddum Comme je n'avais pas de retour direct (d'habitude je peux utiliser le fichier de trad directement dans son milieu d'origine, comme avec un CMS) j'ai gardé la même nomenclature (donc les underscore) et certains termes n'étant pas forcément clair en anglais sur leur position (je ne retrouvais pas leur équivalant dans l'appli) je me suis dit on va faire un premier jet et on corrigera. 

Du coup j'y vais :)

**Comment by XITRIX** [comment]
Will it be possible to translate "Thanks" section also?

**Comment by Cervoz** [comment]
Yes, I just haven't had time to go over everything since the last time, but it's planned!

**Comment by Olfdev** [comment]
How can I take the French translation over? 
When I search for French in POEditor, nothing appears.

I'd like to clean the underscores and correct some out of context translations.

Thanks!

**Comment by XITRIX** [comment]
@Olfdev, Take a look again pls, I've added French into POEditor

**Comment by Olfdev** [comment]
Thanks, I can see french now, just waiting to be approved and I'll work on it.
Thanks!

**Comment by XITRIX** [comment]
<img width="1305" alt="image" src="https://user-images.githubusercontent.com/9553519/210050484-98bdd3a0-2eb8-4073-9450-226e840f6712.png">
Looks like you already was approved

**Comment by Olfdev** [comment]
Mhmm weird because of I click on the flag, I get this message and I can't do anything:
![Screenshot_20221230-142859](https://user-images.githubusercontent.com/80658604/210075389-ac0536cc-2892-4b2d-a31a-ccccb8f3e81c.png)

**Comment by XITRIX** [comment]
@Olfdev, omg, sorry, what a stupid interface poeditor has. I had to open detailed information about your account, and only there I could accept your request. Now you should be able to edit French localisation

**Comment by Olfdev** [comment]
@XITRIX Sorry it took so long but the French translation is done. The previous translator forgot to select English as language reference so he translated the StringIDs.
Feel free to integrate it whenever you want!

**Comment by XITRIX** [comment]
@Olfdev, thanks for you work, I've updated localization, you could check it [here](https://github.com/XITRIX/Moonlight-Switch/actions/runs/4408831702)

**Comment by Olfdev** [comment]
@XITRIX Thanks for the update! I checked and there is some issues:
1. Some Strings are missing in POEditor for translation:
https://img.super-h.fr/images/2023/03/17/31ae22efbf0956ddfa94eabd54a2f944.jpg
https://img.super-h.fr/images/2023/03/17/e7284194d01eaf6dd07a355bd225d2d0.jpg

"Keyboard" (not the same as "keyboard" in context "mouse_input" which is available in POEditor)
"Keyboard type"
"Full-sized"
"Compact"
"Touchscreen mode"
"Timeout was reached"

1. Some vertical choices are overflowing at the bottom. This is not translation related as it also happens on the keyboard type choice screen (which has no translation available as stated in 1.) but still, it doesn't happen with default English language:
https://img.super-h.fr/images/2023/03/17/d58c686334fd4c00d1857d8ab17db9e6.jpg
https://img.super-h.fr/images/2023/03/17/a313803d54ee50a39f7de1e79f64f7a8.jpg
https://img.super-h.fr/images/2023/03/17/1a77370f5dd77cabc1770576536d7982.jpg
https://img.super-h.fr/images/2023/03/17/5a06beff41b0d2b3c91a17e3ed6921c8.jpg
https://img.super-h.fr/images/2023/03/17/12c9891bbb66a21a2b8f7e4d467f2d8c.jpg

Thanks!
